tsconfig.json

```diff
@@ -10,6 +10,9 @@
     "noEmit": true,
     "forceConsistentCasingInFileNames": true,
     "noUnusedLocals": true,
+    "paths": {
+      "*": ["./src/types/*"]
```

Comment:
Needed to resolve the `.d.ts` file with type declarations for the new dependency